### PR TITLE
feat(server): fixing double summary emails per week

### PR DESCRIPTION
**Pull request by fabis94**

### CODE REVIEW

File: packages/server/modules/core/repositories/scheduledTasks.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  return task ?? null
+}
+
+export async function saveScheduledTask(
+  scheduledTask: ScheduledTaskRecord
+): Promise<ScheduledTaskRecord> {
+  const [record] = await ScheduledTasks.knex<ScheduledTaskRecord>()
+    .insert(scheduledTask)
+    .onConflict(['createdAt', 'taskName'])
+    // this puts the full table name + col name into string, which doesn't work here
```
Comment (fabis94) [inline packages/server/modules/core/repositories/scheduledTasks.ts:22]: you can use `ScheduledTasks.withoutTablePrefix` here

Comment (gjedlicska) [reply packages/server/modules/core/repositories/scheduledTasks.ts:22]: nice, it was giving me a headache without that

**Comment by gjedlicska** [comment]
currently this includes a cron trigger of 1000 seconds for weekly activity digests, this should be rolled back before release

**Comment by gjedlicska** [comment]
fixes #1064

**Review by fabis94 — COMMENTED** [review]
@gjedlicska approved (can't approve my own PR)

**Comment by gjedlicska** [comment]
don't worry, i'll approve it for you :D

**Review by gjedlicska — DISMISSED** [review]

**Review by gjedlicska — APPROVED** [review]

**Pull request merged**